android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -419,8 +420,8 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?) : ReactCont
   }
 
   private fun tryInitializeHandlerForReactRootView(ancestorViewTag: Int) {
-    val uiManager = reactApplicationContext.getNativeModule(UIManagerModule::class.java)
-    val rootViewTag = uiManager!!.resolveRootTagFromReactTag(ancestorViewTag)
+    val uiManager = reactApplicationContext.UIManager
+    val rootViewTag = uiManager.resolveRootTagFromReactTag(ancestorViewTag)
```

Comment:
IMO We don't need `uiManager` variable here.

Reply:
It's used a second time below. I don't know what overhead does `ReactContext.getNativeModule` have, but I think calling it once is better even if it hurts code readability slightly.

Reply:
Ah, sure, didn't see the second use.